HoloPipelines/pipelines/components/getMultipleNumpy.py

```diff
@@ -0,0 +1,50 @@
+#TODO change file name cos i legit dont know what to name this. File naming is not the same to the rest of comp format anyway
```

Comment:
Renamed to ?  HoloPipelines/pipelines/components/compSeparateNumpy.py ` in Pap's branch

Reply:
deleted